posts/2019/05/quansight-partners-with-quansight-to-sponsor-spyder.md

```diff
@@ -0,0 +1,112 @@
+<!--
+.. title: Quansight Partners with Quansight to sponsor Spyder
+.. slug: quansight-partners-with-quansight-to-sponsor-spyder
+.. date: 2019-05-16 21:22:58 UTC-05:00
+.. tags: Spyder
+.. category: 
+.. link: 
+.. author: Gonzalo Peña-Castellanos
+.. description: 
+.. type: text
+-->
+
+TDK-Micronas is sponsoring Spyder development efforts through Quansight Labs.
+This will enable the development of some features that have been requested by
+our users, as well as new features that will help TDK develop custom Spyder
+plugins in order to complement their Automatic Test Equipment (ATE’s) in the
+development of their Application Specific Integrated Circuits (ASIC’s).
+
+## Multiple Projects aka Workspaces
+
+In the current state Spyder can only handle one active project at a time.
+Although in the past we had basic support for workspaces, it was never a fully
+functional feature, so to ease development and simplify the user experience,
+we decided to remove it on the 3.x series.
+
+For TDK-Micronas and within its ATE tooling development as well as for a large
+part of the Spyder user base, the use of multiple simultaneous projects is a
+common pattern, so this enhancement will:
+
+* Move from single to multiple active projects concept
+* The name `Project Explorer` would then become `WorkSpace Explorer` and
+copying from one project to another is done in the 'Explorer' by means of
+context menus. 
+
+It is worth mentioning that projects are considered an “Advanced feature”,
+so it will not be forced on users that just want to get some work done.
+Projects usage is, and will remain, and optional feature.
+
+<!-- TEASER_END -->
+
+## Project Types
+
+Since the revamp of Spyder Projects with the 3.x series, the new dialog hinted
+the possibility of creating different types of projects. Currently spyder
+provides an empty project type.
+
+![Project creation](/images/spyder-project-creation.png)
+ 
+This feature will:
+* Expand the (current) Project Explorer API to handle Project Types.
+* Define and implement the minimal aspects that should be extensible by a
+Project Type
```

Comment:
this isn't rendered as a list, needs blank lines around it and an indent for the last line